lib/common.js

```diff
@@ -88,6 +91,8 @@ var mergeChunks = function(chunks) {
 
   if(!areBuffers) {
     //  When the chunks are not buffers we assume that they are strings.
+    // This line is not covered. In reality is this ever called with strings
+    // instead of buffers?
```

Comment:
shall we prefix these with `TODO:`? Sounds like something we should get back to? We probably will anyway as we update the code base :)

Reply:
Sure, makes sense!